system/Database/SQLSRV/Connection.php

```diff
@@ -120,6 +120,10 @@ public function connect(bool $persistent = false)
             unset($connection['UID'], $connection['PWD']);
         }
 
+        if (strpos($this->hostname, ',') === false) {
```

Comment:
if can be combined so no need to re-assign hostname when no port:

```php
if (strpos($this->hostname, ',') === false && $this->port !== '')

Reply:
I agree. Thanks